suites/pacific/rados/sanity_rados.yaml

```diff
@@ -0,0 +1,69 @@
+# Test suite to deploy a minimal RHCS Ceph Storage Cluster which can then be used to
+# build out the environment.
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_deployment.py
+      polarion-id:
+      config:
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                registry-url: registry.redhat.io
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
```

Comment:
Can we remove this test for the time being? There is a patch set submitted by Sunil that would support retrieval of the cluster status. We could use that here.

Also verify is a misnomer here as we only get the status and no evaluation is done on the retrieved data.

Reply:
Removed the test